### PR TITLE
DockerfileLintBear.py: Add SEE_MORE attribute

### PR DESCRIPTION
**Pull request by harshhx17**
This adds the official source link to the SEE_MORE attribute

Closes coala#2287
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by harshhx17** [comment]
Thanks sangamcse and kriti21 😁

File: bears/configfiles/DockerfileLintBear.py [hunk 1]
```diff
@@ -22,6 +22,8 @@ class DockerfileLintBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax', 'Smell'}
+    SEE_MORE = ('https://github.com/projectatomic'
+                + '/dockerfile_lint#dockerfile-lint')
```
Comment (anctartica) [inline bears/configfiles/DockerfileLintBear.py]: can you have this on a single line?

Comment (harshhx17) [reply bears/configfiles/DockerfileLintBear.py]: I had tried that first but that would make the line of 81 characters which will not follow PEP8 styles
I also tried using
# Ignore PycodestyleBear (E501)
    SEE_MORE = 'https://github.com/projectatomic/dockerfile_lint#dockerfile-lint'
on a different branch on my fork...and tested it with these bots...
Its giving the same errors :disappointed:

**Comment by harshhx17** [comment]
Thanks for the help!! @alphadose 
But that ain't working either :disappointed: 
https://circleci.com/gh/harshhx17/coala-bears/6?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: bears/configfiles/DockerfileLintBear.py (hunk 1 above)
Comment (sangamcse) [reply bears/configfiles/DockerfileLintBear.py]: @anctartica, then bot will catch warning of line exceeding 72 characters 😅

Comment (sangamcse) [reply bears/configfiles/DockerfileLintBear.py]: Please use curly bracket @harshhx17. And I think https://github.com/projectatomic/dockerfile_lint this much link is enough.

**Comment by harshhx17** [comment]
@sangamcse  I have tried using curly braces(on my fork) as mentioned by you and Anish..
But the circleci is still giving errors...
Here is the link to the error logs and the commit
Error log - https://circleci.com/gh/harshhx17/coala-bears/6?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
Link to the commit - https://github.com/harshhx17/coala-bears/commit/958d348a088b6719305b84a00a0b8a542624c83e
Thanks for replying :wink:

**Comment by sangamcse** [comment]
Then better use `SEE_MORE = 'https://github.com/projectatomic/dockerfile_lint'`. This will resolve all checks.

**Comment by harshhx17** [comment]
Thanks @sangamcse.... :smile: 
But that ain't working either :disappointed: 
The circleci logs link - https://circleci.com/gh/harshhx17/coala-bears/6?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
The github commit link in my fork - https://github.com/harshhx17/coala-bears/commit/295fd996dc8fb6be826295f828f0546a996e7275

**Comment by sangamcse** [comment]
Just give a push to this PR after requested changes are done and then I will check it from here.

**Comment by anctartica** [comment]
yes please push your changes so that we can tell what is going wrong

**Comment by harshhx17** [comment]
I am sorry for the inconvenience... 
I will do that immediately :wink:

**Comment by kriti21** [comment]
@harshhx17 I think your builds are failing because of some problem with sourceforge through no fault of yours. @Makman2 Please see the logs and if I am right, you may as well start the rebuild process :)

**Comment by harshhx17** [comment]
Okk!! Thanks for the quick review.
coala is awesome 😊

**Comment by Makman2** [comment]
ack cc61377e0068c49759a23ee28fd1c405a5092338

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@harshhx17 CircleCI screwed again and I can't manually restart the builds. Could you regenerate the commit hash by executing `git commit --amend --no-edit && git push -f`?

**Comment by harshhx17** [comment]
The build is still failing with the same logs @Makman2 
Plzz... have a look :smiley:

**Comment by Makman2** [comment]
ack b305809e9f85b597aadcb795cc79cb5887d45e10

**Comment by Makman2** [comment]
Yeah this commit shouldn't fail in CI... we have to fix that asap^^

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by harshhx17** [comment]
This pr is currently being built under harshhx17/coala-bears in circleci(I have enabled circleci on my fork so that I can test configuration for circleci 2.0)...
The opam dependencies are failing on my fork, However since the build for that is already cached in coala/coala-bears, it is passing here...(Should I open an issue for it)
However, the builds(tests) are still not passing for builds that are under coala/coala-bears..
I can force push with  circleci disabled on my fork(will initiate build under coala/coala-bears), anytime it is required..(Just inform me :wink:)

**Comment by harshhx17** [comment]
@jayvdb ^
I think the builds will pass if I do that since the dockerfilelintbear test has been fixed :wink:

**Comment by jayvdb** [comment]
> I can force push with circleci disabled on my fork(will initiate build under coala/coala-bears), anytime it is required..(Just inform me wink)

yes please.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2292/commits/80b79bad15ceab458ff259d5b969b0b9ee7b138e

**Comment by harshhx17** [comment]
Done @jayvdb :smile:

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
gah!  Your circle ci hasnt been disabled!

**Comment by harshhx17** [comment]
I am sorry @jayvdb ....
I had disabled it for this build and then re-enabled it to test https://github.com/coala/coala-bears/pull/2312
I will keep it disabled for now until this pr is merged :smile: 
Btw, I don't think this build should fail on my fork for no reason...
Its passing under coala-bears because the cache for the file is stored...
I think it will fail even on coala-bears if the cache is flushed 
Should I open an issue for it because once we switch to circleci 2.0 the new cache will be generated and opam dependencies will fail once there

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**